<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotoEditor</class>
 <widget class="QMainWindow" name="PhotoEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PhotoEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="horizontalFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>361</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPawel_Grabowski_CG">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Image"/>
    <addaction name="actionSave_Image"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuFunctional_Filters">
     <property name="title">
      <string>Functional Filters</string>
     </property>
     <addaction name="actionInversion"/>
     <addaction name="actionBrightness"/>
     <addaction name="actionContrast"/>
     <addaction name="actionGamma"/>
    </widget>
    <widget class="QMenu" name="menuConvolution_Filters">
     <property name="title">
      <string>Convolution Filters</string>
     </property>
     <addaction name="actionBox_Blur"/>
     <addaction name="actionGaussian_Blur"/>
     <addaction name="actionSharpen"/>
     <addaction name="actionEdge_Detection"/>
     <addaction name="actionEmboss"/>
    </widget>
    <widget class="QMenu" name="menuDither_Quantize">
     <property name="title">
      <string>Dither &amp; Quantize</string>
     </property>
     <addaction name="actionOrdered_Dithering"/>
     <addaction name="actionUniform_Quantization"/>
    </widget>
    <addaction name="menuFunctional_Filters"/>
    <addaction name="menuConvolution_Filters"/>
    <addaction name="menuDither_Quantize"/>
    <addaction name="actionCustom_Filter"/>
    <addaction name="actionLaboratory"/>
    <addaction name="actionReset"/>
    <addaction name="actionB_W"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionDescription_Website"/>
   </widget>
   <addaction name="menuPawel_Grabowski_CG"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Image">
   <property name="text">
    <string>Load Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionDescription_Website">
   <property name="text">
    <string>Description Website</string>
   </property>
  </action>
  <action name="actionInversion">
   <property name="text">
    <string>Inversion </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionBrightness">
   <property name="text">
    <string>Brightness</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionContrast">
   <property name="text">
    <string>Contrast</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionGamma">
   <property name="text">
    <string>Gamma</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actionSave_Image">
   <property name="text">
    <string>Save Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionBox_Blur">
   <property name="text">
    <string>Box Blur</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+5</string>
   </property>
  </action>
  <action name="actionGaussian_Blur">
   <property name="text">
    <string>Gaussian Blur</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+6</string>
   </property>
  </action>
  <action name="actionSharpen">
   <property name="text">
    <string>Sharpen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+7</string>
   </property>
  </action>
  <action name="actionEdge_Detection">
   <property name="text">
    <string>Edge Detection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+8</string>
   </property>
  </action>
  <action name="actionEmboss">
   <property name="text">
    <string>Emboss</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+9</string>
   </property>
  </action>
  <action name="actionCustom_Filter">
   <property name="text">
    <string>Custom Filter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionLaboratory">
   <property name="text">
    <string>Laboratory 1</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionOrdered_Dithering">
   <property name="text">
    <string>Ordered Dithering</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionUniform_Quantization">
   <property name="text">
    <string>Uniform Quantization</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionB_W">
   <property name="text">
    <string>B&amp;W</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
